src/Parser/Shaman/Restoration/Modules/Spells/GiftOfTheQueen.js

```diff
@@ -45,29 +50,60 @@ class GiftOfTheQueen extends Analyzer {
       giftOfTheQueenCBTFeeding += this.cooldownThroughputTracker.cbtFeed[SPELLS.GIFT_OF_THE_QUEEN_DUPLICATE.id].healing;
     }
     
-    const giftOfTheQueenCBTFeedingPercent = giftOfTheQueenCBTFeeding / (giftOfTheQueenRawHealing + giftOfTheQueenDuplicateRawHealing);
+    return giftOfTheQueenCBTFeeding / (giftOfTheQueenRawHealing + giftOfTheQueenDuplicateRawHealing);
     
-    when(giftOfTheQueenTargetEfficiency).isLessThan(0.95)
+  }
+
+  suggestions(when) {
+    const hasCBT = this.combatants.selected.hasTalent(SPELLS.CLOUDBURST_TOTEM_TALENT.id);
+   
+    when(this.giftOfTheQueenTargetEfficiency()).isLessThan(0.95)
       .addSuggestion((suggest, actual, recommended) => {
         return suggest(<span>Try to always cast <SpellLink id={SPELLS.GIFT_OF_THE_QUEEN.id} /> at a position where both the initial hit and the echo from <SpellLink id={SPELLS.DEEP_WATERS.id} /> will hit all 6 potential targets.</span>)
           .icon(SPELLS.GIFT_OF_THE_QUEEN.icon)
-          .actual(`${formatPercentage(giftOfTheQueenTargetEfficiency)} % of targets hit`)
+          .actual(`${formatPercentage(this.giftOfTheQueenTargetEfficiency())} % of targets hit`)
           .recommended(`> ${formatPercentage(recommended)} % of targets hit`)
           .regular(recommended - .05).major(recommended - .15);
       });
     
+    const feedingPercent = this.giftOfTheQueenCBTFeedingPercent();
     if (hasCBT) {
-      when(giftOfTheQueenCBTFeedingPercent).isLessThan(0.85)
+      when(feedingPercent).isLessThan(0.85)
         .addSuggestion((suggest, actual, recommended) => {
           return suggest(<span>Try to cast <SpellLink id={SPELLS.GIFT_OF_THE_QUEEN.id} /> while <SpellLink id={SPELLS.CLOUDBURST_TOTEM_TALENT.id} /> is up as much as possible.</span>)
             .icon(SPELLS.GIFT_OF_THE_QUEEN.icon)
-            .actual(`${formatPercentage(giftOfTheQueenCBTFeedingPercent)} % of GotQ healing fed into CBT`)
+            .actual(`${formatPercentage(feedingPercent)} % of GotQ healing fed into CBT`)
             .recommended(`> ${formatPercentage(recommended)} % of GotQ healing fed into CBT`)
             .regular(recommended - .2).major(recommended - .4);
         }); 
       }   
   }
 
+  get giftOfQueenTargetEfficiencySuggestionThreshold(){
```

Comment:
I recommend also using these threshold values in the `suggestions()` method so that you have 1 place you need to update should you ever want to change these. Also try to use the `actual` property there if possible.